From aa56dd948bf3da2066955fc209182615c96f7793 Mon Sep 17 00:00:00 2001
From: Divya Narayanan Poojary <dnaray@codeaurora.org>
Date: Mon, 3 Jul 2017 11:26:29 +0530
Subject: [PATCH 4/5] tinycompress: Generate vendor specifc tinycompress

Generate libtinycompress_vendor that contains the
tinycompress extensions

CRs-Fixed: 2074491

Change-Id: I1466c13f216ce723d63155bc099eccad4fdccdfe
---
 compress.c                          | 2 ++
 include/tinycompress/tinycompress.h | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/compress.c b/compress.c
index 8fa1386..1ab2afd 100644
--- a/compress.c
+++ b/compress.c
@@ -572,6 +572,7 @@ int compress_set_gapless_metadata(struct compress *compress,
 	return 0;
 }
 
+#ifdef USE_VENDOR_EXTN
 int compress_set_next_track_param(struct compress *compress,
 	union snd_codec_options *codec_options)
 {
@@ -582,6 +583,7 @@ int compress_set_next_track_param(struct compress *compress,
 		return oops(compress, errno, "cannot set next track params\n");
 	return 0;
 }
+#endif
 
 bool is_codec_supported(unsigned int card, unsigned int device,
 		unsigned int flags, struct snd_codec *codec)
diff --git a/include/tinycompress/tinycompress.h b/include/tinycompress/tinycompress.h
index f88c2a2..b7e57bc 100644
--- a/include/tinycompress/tinycompress.h
+++ b/include/tinycompress/tinycompress.h
@@ -246,8 +246,10 @@ int compress_set_gapless_metadata(struct compress *compress,
  * @codec_options: codec options of compress stream based on codec type
  */
 
+#ifdef USE_VENDOR_EXTN
 int compress_set_next_track_param(struct compress *compress,
 			union snd_codec_options *codec_options);
+#endif
 
 /*
  * is_codec_supported:check if the given codec is supported
-- 
2.17.1

